.chronus/config.yaml

```diff
@@ -55,13 +55,17 @@ versionPolicies:
       - "@typespec/sse"
       - "@typespec/streams"
       - "@typespec/xml"
+  - name: python-client
+    type: independent
+    step: patch
```

Comment:
```suggestion
```

step is for lockStep only

Reply:
looking at the previous policy, it has `type: lockstep` and `step: minor` so I'm kind of confused

Reply:
I went with `step: patch` instead of `step: minor` since I don't want to only officially release minor versions, but let me know if my understanding here isn't right

Reply:
if you use lockStep policy it means all package in the policy will always increase by the given step. For independant it will just respect the change description(if you do major, minor, patch,etc.)